pages/auth/quiz.js

```diff
@@ -0,0 +1,38 @@
+import Head from "next/head";
```

Comment:
for easier navigation and better logic, i put all the pages for our main content to go under the `auth` folder.  NextJS has some nice nested router handling so for example:
`/pages/auth/index.js` == `localhost:3000.com/auth`
`/pages/auth/quiz.js` == `localhost:3000.com/auth/quiz`
etc...